common/src/main/java/gov/cms/ab2d/common/util/FileUtil.java

```diff
@@ -0,0 +1,35 @@
+package gov.cms.ab2d.common.util;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
+public class FileUtil {
+
+    @VisibleForTesting
+    public static void createTmpFileForDownload(String jobID, String fileName) throws IOException {
+        List<String> lines = List.of("{",
```

Comment:
I was using static files, but changed to using this just so it can be created in the directory that is based off a job id, something that can't be predicted since it will be a UUID. It seems to mimic production more closely by doing it that way.

Reply:
You'd have that test file under `src/test/resource` and in your test you'd actually copy it into whatever path you need. We don't really to maintain the file's content within the code itself.

Reply:
@adaykin We can probably just drop this class altogether. Instead, in each test class we could just copy the `src/test/resource/test.ndjson` to wherever we need it.